e2e-tests/users-app/run

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+set -o errexit
+
+compare() {
+	local database="$1"
+	local command="$2"
+	local uri="$3"
+	local target="$4"
+
+	run_mongo "use ${database}\n ${command}" "$uri" "mongodb" \
+		| egrep -v 'I NETWORK|W NETWORK|F NETWORK|Error saving history file|Percona Server for MongoDB|connecting to:|Unable to reach primary for set|Implicit session:|versions do not match|Error saving history file:' \
+		| $sed -re 's/ObjectId\("[0-9a-f]+"\)//; s/-[0-9]+.svc/-xxxesvc/' \
+		| sed '/"userId"/d' \
+			>$tmp_dir/${target}
+
+	diff ${test_dir}/compare/${target}.json $tmp_dir/${target}
+}
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+set_debug
+
+psmdb="some-name"
+cluster="some-name-rs0"
+
+create_infra $namespace
+
+desc 'create secrets and start client'
+kubectl_bin apply -f "${conf_dir}/client.yml" \
+	-f "${conf_dir}/secrets.yml" \
+	-f "${conf_dir}/minio-secret.yml"
+
+
+desc "create first PSMDB cluster $cluster"
+apply_cluster $test_dir/conf/$cluster.yml
+
+
+desc 'Check if all 3 Pods started'
+wait_for_running $cluster 3
+
+
+desc 'check user created on cluster creation'
+kubectl_bin apply -f "${test_dir}/conf/app-user-secrets.yml"
```

Comment:
Why do we apply secret after starting cluster? To check that it stars ok if the secrets were not provided?

Reply:
No particular reason, we can apply those before cluster creation, does not matter. But I'll create them before, it will be less confusing maybe.